lcci/16.01.Swap Numbers/README.md

```diff
@@ -89,6 +89,17 @@ function swapNumbers(numbers: number[]): number[] {
 }
 ```
 
+```swift
+class Solution {
+    func swapNumbers(_ numbers: inout [Int]) -> [Int] {
+        numbers[0] ^= numbers[1]
```

Comment:
Can you pass test?

![image](https://github.com/doocs/leetcode/assets/117748716/2001b790-b98e-4205-a045-9a6e0d295c2c)